jmeter-java-dsl-csv-random/src/main/java/us/abstracta/jmeter/javadsl/csvrandom/RandomCsvDataSetConfig.java

```diff
@@ -0,0 +1,165 @@
+package us.abstracta.jmeter.javadsl.csvrandom;
+
+import com.blazemeter.jmeter.RandomCSVDataSetConfig;
+import com.blazemeter.jmeter.RandomCSVDataSetConfigGui;
+import org.apache.jmeter.testelement.TestElement;
+import us.abstracta.jmeter.javadsl.core.testelements.BaseTestElement;
+import us.abstracta.jmeter.javadsl.core.testelements.MultiLevelTestElement;
+
+/**
+ * The Random CSV Data Set Config plugin provides the capability to parameterize your Test Plan
+ * from a CSV file in random order of records
+ *
+ * This element uses
+ * <a href="https://github.com/Blazemeter/jmeter-bzm-plugins/blob/master/random-csv-data-set/RandomCSVDataSetConfig.md">
+ * Random CSV Data Set Config</a>, check its documentation for more details.
+ *
+ * By default, this element execute with next params: file encoding is 'UTF-8', delimeter is ',', random order is true,
+ * rewind on the end of list is true and finally ignore first line is true. The defaults came from JMeter.
+ *
+ * @since 0.34
+ */
+public class RandomCsvDataSetConfig extends BaseTestElement implements MultiLevelTestElement {
+
+  private final String filename;
+  private String fileEncoding = "UTF-8";
+  private String delimiter = ",";
+  private String[] variableNames;
+
+  private boolean ignoreFirstLine = false;
+  private boolean randomOrder = true;
+  private boolean rewindOnTheEndOfList = true;
+  private boolean independentListPerThread = false;
+
+
```

Comment:
This multi blank lines should be one of the things that checkstyle should complain about, since in the code conventions we use a setting that avoids unnecessary blank lines